Makefile

```diff
@@ -698,6 +698,11 @@ lint-go: GO_LINT_FLAGS ?=
 lint-go:
 	golangci-lint run -c .golangci.yml --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' $(GO_LINT_FLAGS)
 
+.PHONY: fix-imports
+fix-imports: GO_LINT_FLAGS ?=
+fix-imports:
+	golangci-lint run -c .golangci.yml --disable-all --enable=gci --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' --fix
```

Comment:
Format api/ here too? I assume, from the latest commits, that because it is a separate module golangci is skipping it, is that right?

Reply:
Yes, there is also `build.assets/tooling` that needs to be formatted. I'm adding it now.

Reply:
My 2c: There are a number of mods throughout. If we cover / and /api it's good to start, we can follow up with others in subsequent PRs.